### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by adrianhall**
Streamlined instructions for deploying the reference implementation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
+azd env set DATABASE_PASSWORD "AV@lidPa33word"
+azd env set APP_ENVIRONMENT prod
+azd env set AZURE_LOCATION westus3
+azd env set AZURE_LOCATION2 eastus
+azd up
 ```
 
-If cloning in Windows, you may need to configure support for long paths in your environment depending on how long the folder path is you are cloning into. From the Registry Editor, navigate to HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem and set the DWORD LongPathsEnabled to 1.
+The following walk-through assumes you are using a DevContainer inside Visual Studio Code and walks you through executin
+g the above sequence.
```
Comment (KSchlobohm) [inline README.md:76]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
+azd env new eap-javarwa
+azd env set DATABASE_PASSWORD "AV@lidPa33word"
+azd env set APP_ENVIRONMENT prod
+azd env set AZURE_LOCATION westus3
+azd env set AZURE_LOCATION2 eastus
+azd up
 ```
 
-If cloning in Windows, you may need to configure support for long paths in your environment depending on how long the folder path is you are cloning into. From the Registry Editor, navigate to HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem and set the DWORD LongPathsEnabled to 1.
+The following walk-through assumes you are using a DevContainer inside Visual Studio Code and walks you through executin
```
Comment (KSchlobohm) [inline README.md:75]:
```suggestion
The following walk-through assumes you are using a DevContainer inside Visual Studio Code and walks you through executing the above sequence.
```

File: README.md [hunk 3]
```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
-* [Docker](https://www.docker.com/get-started/)
-* [Permissions to register an application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app)
-* [Dev Containers extension installed in VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+We recommend that you use a DevContainer to deploy this application.  The requirements are as follows:
 
-See [Prequisites](prerequisites.md) for more detailed information about our prerequisites.
+- [Azure Subscription](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+- [Visual Studio Code](https://code.visualstudio.com/).
+- [Docker Desktop](https://www.docker.com/get-started/).
+- [Permissions to register an application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
```
Comment (KSchlobohm) [inline README.md:48]:
```suggestion
- [Permissions to register an application in Azure AD](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
```

File: README.md [hunk 4]
```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
-> the following Azure CLI command.
-> 
-> ```shell
-> az account list-locations --query "[].name" -o tsv
-> ```
+To set the active subscription:
 
-### (Optional) Multi-region support
+```shell
+az accout set --subscription "<your-subscription-id>"
```
Comment (KSchlobohm) [inline README.md:126]:
```suggestion
az account set --subscription "<your-subscription-id>"
```

Comment (KSchlobohm) [inline README.md:126]: Found that when I applied copy and paste as recommended, I had to go back to the portal and get my subscriptionId again.

Would like to see "<your-subscription-id>" assigned to a variable or some other shorthand approach.

File: README.md [hunk 5]
```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
 git clone https://github.com/Azure/reliable-web-app-pattern-java.git
 cd reliable-web-app-pattern-java
-code .
+azd config set alpha.terraform on
+azd env new eap-javarwa
+azd env set DATABASE_PASSWORD "AV@lidPa33word"
+azd env set APP_ENVIRONMENT prod
+azd env set AZURE_LOCATION westus3
+azd env set AZURE_LOCATION2 eastus
+azd up
```
Comment (KSchlobohm) [inline README.md:72]: This approach appears to demonstrate multi-regional provision and single region deploy. The azd up command is unaware of the second resource group.

I think we should choose to deploy the simple version as single region or add the following

> SECONDARY_RESOURCE_GROUP=$(azd env get-values --output json | jq -r .secondary_resource_group)
> azd env set AZURE_RESOURCE_GROUP $SECONDARY_RESOURCE_GROUP
> azd deploy

File: README.md (hunk 2 above)
Comment (KSchlobohm) [inline README.md]: I think it should be `Dev Container` as two words - https://learn.microsoft.com/en-us/shows/beginners-series-to-dev-containers/

**Review by KSchlobohm — COMMENTED** [review]
Added some comments to consider and found a couple of quick fix typos.

I tested the long form set of commands doing single region deployment of non-prod environment. This was successful, I was able to play a video and teardown the environment.

No errors were shown but I had two observations:
1. the `az login` command completed successfully but didn't provide a UX to explain that I could close the window after it was done. I thought it timed out but noticed that the response was sent to the terminal even though the browser showed a blank white screen.
1. the "java developers deploy your code" wrong page was shown so I just waited longer and this refreshed successfully in less than 5 minutes.

File: README.md (hunk 4 above)
Comment (adrianhall) [reply README.md]: Fixed.

File: README.md (hunk 5 above)
Comment (adrianhall) [reply README.md:72]: Fixed.

File: README.md (hunk 2 above)
Comment (adrianhall) [reply README.md]: Fixed throughout.

**Comment by adrianhall** [comment]
Thanks @KSchlobohm - fixed all comments.

**Pull request closed without merging**